src/app/Mappings/components/AddEditMappingModal.tsx

```diff
@@ -167,6 +168,7 @@ const AddEditMappingModal: React.FunctionComponent<IAddEditMappingModalProps> =
                   availableTargets={mappingResourceQueries.availableTargets}
                   builderItems={builderItems}
                   setBuilderItems={setBuilderItems}
+                  isEveryItemFilled={() => builderItems.every((item) => item.source && item.target)}
```

Comment:
It doesn't really matter much, but it's still unnecessary for this to be a function. You can just pass `isEveryItemFilled={builderItems.every((item) => item.source && item.target)}` and change the prop to a boolean, and usages from `isEveryItemFilled()` to `isEveryItemFilled`.

Reply:
Yes, wasn't pay attention there! Thanks